packages/meditrak-app/android/appcenter-gradle.properties

```diff
@@ -0,0 +1,11 @@
+# gradle.properties to be checked in and used by appcenter
+# mustn't contain secrets like the keystore password, but can contain gradle config switches
+
+# AndroidX package structure to make it clearer which packages are bundled with the
+# Android operating system, and which are packaged with your app's APK
+# https://developer.android.com/topic/libraries/support-library/androidx-rn
+android.useAndroidX=true
+# Automatically convert third-party libraries to use AndroidX
+android.enableJetifier=true
+
+org.gradle.jvmargs=-XX\:MaxHeapSize\=1024m -Xmx1024m
```

Comment:
This file will get renamed to `gradle.properties` in appcenter (see the pre-build script)